test/e2e/dist/vault-decryption-chrome.spec.ts

```diff
@@ -130,41 +130,8 @@ async function waitUntilFileIsWritten({
   );
 }
 
-/**
- * Closes the announcements popover if present
- *
- * @param driver
- */
-async function closePopoverIfPresent(driver: Driver) {
```

Comment:
side: this function can be removed as it's not needed anymore. The context about this, is that before, this spec was not run on every PR, and changes on popovers (adding or removing them) cause the spec to fail once it was run in main.

Now we run the spec on every PR, so if a new popover is added or removed, this will be catched at the PR level, so the click button will be added/removed there